### PR TITLE
[Site Request] JPMANGAS #2453

### LINKED ISSUE #2453: [Site Request] JPMANGAS
**Name of the website**
JPMANGAS

**Website urls (examples below)**
- Site: https://www.jpmangas.com/
- Manga List: https://www.jpmangas.com/lecture-en-ligne-list
- Manga example: https://www.jpmangas.com/lecture-en-ligne/samura-comeback
- Chapter online viewer example: https://www.jpmangas.com/lecture-en-ligne/samura-comeback/4.4/10

**Languages**
french

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by Robonau**
resolves #2453
(no icon cause site has none)

### CODE REVIEW

**Comment by 27136773** [comment]
Should add an icon for the website

File: src/web/mjs/connectors/JpMangas.mjs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class JpMangas extends MangaReaderCMS {
+
+    constructor() {
+        super();
+        super.id = 'jpmangas';
+        super.label = 'JpMangas';
```
Comment (27136773) [inline src/web/mjs/connectors/JpMangas.mjs]: May use same spelling as on website

image

Comment (Robonau) [reply src/web/mjs/connectors/JpMangas.mjs]: im either blind or that is the same spelling

Comment (Robonau) [reply src/web/mjs/connectors/JpMangas.mjs]: figured it out

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**